scripts/eject.js

```diff
@@ -104,6 +104,11 @@ prompt('Are you sure you want to eject? This action is permanent. [y/N]', functi
   });
   delete hostPackage.scripts['eject'];
 
+  // explicitly specify ESLint config path for editor plugins
+  hostPackage.eslintConfig = {
+    extends: "./config/eslint.js",
```

Comment:
I might be totally wrong, but is it safe to use `/` since it's possible to be `\` on windows? I think `path.resolve` works cross platform.